### PR TITLE
fix: Max LB name length

### PR DESCRIPTION
**Pull request by wandb-zacharyblasczyk**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -184,6 +184,11 @@ module "redis" {
   kms_key_arn = local.kms_key_arn
 }
 
+locals {
+  max_lb_name_length = 32 - length("-alb-k8s")
+  lb_name_truncated  = "${substr(var.namespace, 0, min(length(var.namespace), local.max_lb_name_length))}-alb-k8s"
```
Comment (jsbroks) [inline main.tf:189]:
```suggestion
  lb_name_truncated  = "${substr(var.namespace, 0, local.max_lb_name_length)}-alb-k8s"
```
I don't think we need the min

**Review by gls4 — CHANGES_REQUESTED** [review]
Let's simply set the name of the ALB to the namespace (or a substring thereof) and be done with it.

**Comment by jsbroks** [comment]
@gls4 we cannot because that is already used by the other lb

**Review by jsbroks — APPROVED** [review]

**Comment by jsbroks** [comment]
This PR is included in version 4.1.2 :tada:

**Pull request merged**